app/controllers/guidances_controller.rb

```diff
@@ -42,9 +42,18 @@ def admin_create
       guidance_params[:theme_ids].map{|t| guidance.themes << Theme.find(t.to_i) unless t.empty? }
     end
 
+    guidance_group = GuidanceGroup.where(id: @guidance.guidance_group_id).first
```

Comment:
you should be able to switch this to `GuidanceGroup.find(@guidance.guidance_group_id)`. Its more of a preference though so ok to leave as is for now.

Reply:
when a guidance_group_id doesn't exist for the particular guidance, the find method throws nil error and doesn't proceed to the unless statement at all. Whereas the where condition returns an empty active relation and proceeds without the exception. 
Thus, have been using the where clause to prevent nil exception whenever need to succeed to a conditional loop (mostly coding preference, open to suggestions !)